conf/sample.ini

```diff
@@ -446,6 +446,16 @@ log_queries =
 # This limit will protect the server from render overloading and make sure notifications are sent out quickly
 ;concurrent_render_limit = 5
 
+
+## Default setting for alert calculation timeout
```

Comment:
Should only be 1 #-character, not two. Two other places below with this problem.